set.mm

```diff
@@ -645688,8 +645842,10 @@ number axioms (add ~ ax-10 , ~ ax-11 , ~ ax-13 , ~ ax-nul , and remove
     exp11d.3 $e |- ( ph -> N e. ZZ ) $.
     exp11d.4 $e |- ( ph -> N =/= 0 ) $.
     exp11d.5 $e |- ( ph -> ( A ^ N ) = ( B ^ N ) ) $.
-    $( ~ sq11d for positive real bases and nonzero exponents.  (Contributed by
-       Steven Nguyen, 6-Apr-2023.) $)
+    $( ~ sq11d for positive real bases and nonzero exponents.  The base cannot
+       be generalized much further, since if ` N ` is even then we have
+       ` A ^ N = -u A ^ N ` .  TODO-SN: Avoid ~ df-cxp .  (Contributed by SN,
+       6-Apr-2023.) $)
```

Comment:
What **can** be generalized to complex bases, on the other hand, is the conclusion of equality of the absolute values.  If you are interested, make yourself a TODO!